zulipterminal/core.py

```diff
@@ -161,12 +161,12 @@ def maximum_popup_dimensions(self) -> Tuple[int, int]:
                                  self.loop.screen.get_cols_rows())
         return max_cols, max_rows
 
-    def show_pop_up(self, to_show: Any) -> None:
+    def show_pop_up(self, to_show: Any, category: str='msg_related') -> None:
```

Comment:
Could you explain the use of the default value here?

Reply:
Yeah, that's not needed, sorry for confusion :sweat_smile: